src/main/java/org/folio/dao/budget/BudgetExpenseClassPostgresDAO.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.dao.budget;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Tuple;
+import org.folio.rest.jaxrs.model.BudgetExpenseClass;
+import org.folio.rest.persist.DBClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.folio.rest.util.ResponseUtils.handleFailure;
+
+public class BudgetExpenseClassPostgresDAO implements BudgetExpenseClassDAO{
+
+  public Future<List<BudgetExpenseClass>> getBudgetExpenseClasses(String sql, Tuple params, DBClient client) {
```

Comment:
Why is this method accepting raw sql and parameters as input? I would expect that as a DAO, it has a sql query encapsulated within. With this interface, it allows querying from a database table that is not for expense classes.